third_party/amd/backend/compiler.py

```diff
@@ -36,6 +36,14 @@ def get_gfx9_limits(lhsType, rhsType):
     return lambda lhsType, rhsType: (1, 1, 1) if is_fma_supported(lhsType.scalar, rhsType.scalar) else (16, 16, 16)
 
 
+def is_pingpong_enabled(arch):
+    if arch == 'gfx942':
```

Comment:
Nit: Actually I feel something like the following would be simpler:

```python
default = "1" if arch == "gfx942" else "0"
return os.getenv("TRITON_HIP_USE_BLOCK_PINGPONG", default) == "1"
```

Reply:
done